src/Pyz/Yves/CheckoutPage/Theme/default/views/payment/payment.twig

```diff
@@ -79,7 +79,7 @@
                     } only %}
                         {% block fieldset %}
                             <div class="js-payment-method-{{ embed.index }} {% if embed.index != '1-0' %} is-hidden{% endif %}">
-                                <h2 class="title title--primary">{{ embed.toggler.vars.label | trans }}</h2>
+                                <h2 class="title title--primary {% if embed.toggler.vars.value matches '#foreignPayments\[[a-zA-Z0-9_-]+\]#' %} is-hidden{% endif %}">{{ embed.toggler.vars.label | trans }}</h2>
```

Comment:
Why do we need to render a hidden title?
Can `embed.toggler.vars.value matches '#foreignPayments\[[a-zA-Z0-9_-]+\]#'` be moved to the BE layer and send a boolean to the template?

Reply:
@FRITS-Kh this problem doesn't exist in Suite, so it's only an issue in b2c design, so one more file (PHP controller) should be overwritten on the project level, which I would love to avoid.

Reply:
So please add a variable for the condition to increase readability.
And avoid the title rendering instead of hidding.